.github/workflows/performance.yml

```diff
@@ -0,0 +1,36 @@
+name: Performance Tests
+
+on:
+  push:
+    branches:
+      - trunk
+      - 'feature/**'
+
+jobs:
+  performance:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == '10up/wordpress-develop' || github.event_name == 'push' }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Docker debug information
```

Comment:
The end-to-end tests start the environment before printing debug info so it can also show info about PHP/MySQL running in the container, which might be nice to add.

Reply:
Thanks @joemcgill. Added requested details.